packages/mux-video/src/index.ts

```diff
@@ -831,7 +798,13 @@ class MuxVideoBaseElement extends CustomVideoElement implements Partial<MuxMedia
         if (newValue) {
           fetch(newValue)
             .then((resp) => resp.json())
-            .then((json) => (this.metadata = json))
+            .then((json) => {
+              this.metadata = json;
+              const metadata = json?.[0]?.metadata ?? [];
+              const planMeta = metadata.find((m: { value: string }) => m.value === 'mux-free-plan');
+
+              if (planMeta) this.logo = 'default';
```

Comment:
don't set `this.logo` here but an internal state property like `this.#logo`. 
make the getter fallback to `this.#logo` if the logo attribute is not set.

```js
this.#logo = ''
this.updateLogo()
```